index.html

```diff
@@ -1,10 +1,115 @@
-<!DOCTYPE html>
-<html>
-<head>
-    <link rel="stylesheet" content="text/css" href="./style/fullscreen.css">
-    <script src="./js/phaser.js"></script>
-    <script src="./js/Create.js"></script>
-    <script src="./js/Preload.js"></script>
+<!doctype html> 
+<html lang="en"> 
+<head> 
+    <meta charset="UTF-8" />
+    <title>Making your first Phaser 3 Game - Part 10</title>
+    <script src="//cdn.jsdelivr.net/npm/phaser@3.11.0/dist/phaser.js"></script>
```

Comment:
Can you please replace this with the Phaser.js file ./js/lib as we need this to make the game run in the browser